### PR TITLE
Connect/disconnect gw on update [SR- 1681]

### PR DESCRIPTION
**Pull request by sarthakJsonar**

### CODE REVIEW

File: dsfhub/resource_log_aggregator.go [hunk 1]
```diff
@@ -816,6 +816,14 @@ func resourceLogAggregatorUpdate(d *schema.ResourceData, m interface{}) error {
 	serverType := d.Get("server_type").(string)
 	createResource(&logAggregator, serverType, d)
 
+	log.Printf("[DEBUG] AuditPullEnabled value in logAggregator is: '%v'", logAggregator.Data.AssetData.AuditPullEnabled)
```
Comment (mattJsonar) [inline dsfhub/resource_log_aggregator.go]: Would this make more sense to print _after_ we set the value? Or perhaps, before and after?

**Review by mattJsonar — DISMISSED** [review]

File: dsfhub/resource_log_aggregator.go (hunk 1 above)
Comment (sarthakJsonar) [reply dsfhub/resource_log_aggregator.go]: Yea I can put one after too. Good suggestion!

**Review by mattJsonar — APPROVED** [review]

**Comment by mattJsonar** [comment]
Not merging until https://onejira.imperva.com/browse/SR-972 is fixed.

**Comment by mattJsonar** [comment]
Handled in https://github.com/imperva/terraform-provider-dsfhub/pull/21, closing.

**Pull request closed without merging**